### DISCUSSION TITLE
Faster Whisper responses?

### DISCUSSION THREAD

**Discussion by aqasemi** [deepgram/community#299] [Feature Requests]
Please have a look at this repo https://github.com/sanchit-gandhi/whisper-jax 
live version https://huggingface.co/spaces/sanchit-gandhi/whisper-jax

Here's a small comparison across the available models for ~14 minutes audio.
>>>
- OpenAI Whisper (T4) | 763s
- faster-whisper (T4)     | 172s
- OpenAI Whisper (API) | 80s
- Whisper JAX (TPU v4-8) | 4.5s
>>>
Source: https://nikolas.blog/making-openai-whisper-faster

**Comment by lukeocodes** [comment]
We made some infrastructural changes aimed at increasing efficiency that had a net negative effect on the quality of service, and we iterated on that to reduce the error rate and improve our response times. 

Whisper is resource intensive, and Deepgram's own models are extremely fast and efficient (see our recent whitepaper). Our infrastructure is optimised for our own models only.
